literalai/observability/step.py

```diff
@@ -549,6 +566,21 @@ def __enter__(self) -> Step:
 
         if active_root_run_var.get() is None and self.step_type == "run":
             active_root_run_var.set(self.step)
+            Traceloop.set_association_properties(
```

Comment:
if otel is not initialized this wont be an issue?

Reply:
Yes it can be safely called even without initialization 👍